handlers/app.go

```diff
@@ -0,0 +1,47 @@
+package handlers
+
+
+import (
+
+	"github.com/knative-scout/app-api/models"
+	"go.mongodb.org/mongo-driver/bson"
+	"net/http"
+	"github.com/gorilla/mux"
+)
+
+// HealthHandler is used to determine if the server is running
+type AppHandler struct {
```

Comment:
Can you rename this handler to something less general? B/c we will have a few more app endpoints.

Maybe name it `GetAppByIDHandler`?